infra-as-code/bicep/modules/policy/assignments/alzDefaults/alzDefaultPolicyAssignments.bicep

```diff
@@ -505,6 +505,13 @@ var varPrivateDnsZonesFinalResourceIds = {
   azureCognitiveSearchPrivateDnsZoneId: '${varPrivateDnsZonesBaseResourceId}privatelink.search.windows.net'
 }
 
+var varPolicyAssignmentScopeName = '${parTopLevelManagementGroupPrefix}${parTopLevelManagementGroupSuffix}'
+var varPolicyExemptionConfidentialOnlineManagementGroup = '${parTopLevelManagementGroupPrefix}-landingzones-confidential-online${parTopLevelManagementGroupSuffix}'
+var varPolicyExemptionConfidentialCorpManagementGroup = '${parTopLevelManagementGroupPrefix}-landingzones-confidential-corp${parTopLevelManagementGroupSuffix}'
```

Comment:
Let's use the existing variables for these as well to keep the code as dry as possible. varManagementGroupIds.landingZonesConfidentialCorp and varManagementGroupIds.landingZonesConfidentialOnline